server/tests/integration_test.rs

```diff
@@ -0,0 +1,998 @@
+        .parent()
+        .expect("Failed to get the parent directory");
+
+    let binary_directory = parent_directory
+        .parent()
+        .expect("Failed to get the directory of the binary");
+
+    binary_directory.to_owned()
+}
+
+/// Start the binary built for integration testing.
+fn start_binary(binary: &str) -> process::Command {
+    // Create path to binary
+    let mut path = get_binary_directory();
+    path.push(binary);
+    path.set_extension(env::consts::EXE_EXTENSION);
+
+    assert!(path.exists());
+
+    // Create command process
+    process::Command::new(path.into_os_string())
+}
+
+/// Start a new Arrow Flight Server to simulate a server for the integration tests.
+fn start_arrow_flight_server() -> Child {
+    let mut process = start_binary("modelardbd")
+        .arg("tests/data")
+        .stdout(process::Stdio::piped())
+        .spawn()
+        .expect("Failed to start Arrow Flight Server");
+
+    let mut line = String::new();
+
+    let mut output = BufReader::new(process.stdout.as_mut().unwrap());
+
+    // Ensure that the process has fully started.
+    while !line.contains("Starting Apache Arrow Flight") {
+        output
+            .read_line(&mut line)
+            .expect("Could not read line of process.");
+    }
+
+    return process;
+}
+
+/// Create a new Arrow Flight client to access the endpoints in the ModelarDB server.
+fn create_flight_service_client(
+    rt: &Runtime,
+    host: &str,
+    port: u16,
+) -> Result<FlightServiceClient<Channel>> {
+    let address = format!("grpc://{}:{}", host, port);
+
+    rt.block_on(async {
+        let fsc = FlightServiceClient::connect(address).await?;
+        Ok(fsc)
+    })
+}
+
+/// Create a model table in the ModelarDB server with Arrow Flight using the `do_action()` SQL Parser.
+fn create_model_table(
+    rt: &Runtime,
+    mut client: &mut FlightServiceClient<Channel>,
+    table_name: String,
+) -> std::result::Result<(), ()> {
+    let cmd = format!(
+        "CREATE MODEL TABLE {}(timestamp TIMESTAMP, value FIELD, tag TAG)",
+        table_name
+    );
+    let action = Action {
+        r#type: "CommandStatementUpdate".to_string(),
+        body: cmd.into_bytes(),
+    };
+
+    rt.block_on(async {
+        let mut _result = client.do_action(Request::new(action)).await;
+
+        Ok(())
+    })
+}
+
+/// Create a normal table without tags in the ModelarDB server with Arrow Flight using the `do_action()` SQL Parser.
+fn create_model_table_without_tags(
+    rt: &Runtime,
+    mut client: &mut FlightServiceClient<Channel>,
+    table_name: String,
+) -> std::result::Result<(), ()> {
+    let cmd = format!(
+        "CREATE MODEL TABLE {}(timestamp TIMESTAMP, value FIELD)",
+        table_name
+    );
+    let action = Action {
+        r#type: "CommandStatementUpdate".to_string(),
+        body: cmd.into_bytes(),
+    };
+
+    rt.block_on(async {
+        let mut _result = client.do_action(Request::new(action)).await;
```

Comment:
Is there any reason to have `let mut _result = `? It seems that it works fine without it.

Reply:
Fixed.